client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/upgrade-popover/index.tsx

```diff
@@ -14,18 +14,25 @@ import './style.scss';
 
 type Props = {
 	context: HTMLElement | null;
+	dismissibleWithPreference?: boolean;
 	isVisible: boolean;
 	position?: string;
-	onClose: () => void;
+	onClose?: () => void;
 };
 
-export default function UpgradePopover( { context, isVisible, position, onClose }: Props ) {
+export default function UpgradePopover( {
+	context,
+	dismissibleWithPreference,
```

Comment:
There are two use cases for the upgrade popover; one does not have to be suppressed when viewed once, like the one we displayed in the feature toggle component.